scripts/data.py

```diff
@@ -0,0 +1,216 @@
+
+server_address = DEFAULT_ADDRESS
+
+
+def root_api_endpoint() -> str:
+    """Return the shared root api endpoint."""
+    return f"http://{server_address}/api"
+
+
+def get_edge_tables(workspace: str) -> List[str]:
+    """Return the edge tables for a given workspace."""
+    resp = requests.get(
+        f"{root_api_endpoint()}/workspaces/{workspace}/tables?type=edge"
+    )
+
+    if resp.ok:
+        tables = json.loads(resp.text)
+        return tables
+
+    return []
+
+
+def get_table_rows(workspace: str, table: str) -> List:
+    """Return the rows of a table."""
+    resp = requests.get(f"{root_api_endpoint()}/workspaces/{workspace}/tables/{table}")
+
+    if resp.ok:
+        rows = json.loads(resp.text)
+        return rows
+
+    return []
+
+
+def check_workspace_exists(workspace: str) -> bool:
+    """Return if the specified workspace exists yet or not."""
+
+    resp = requests.get(f"{root_api_endpoint()}/workspaces/{workspace}")
+
+    if resp.ok:
+        return True
+
+    return False
+
+
+def create_workspace(workspace: str) -> bool:
+    """
+    Create the workspace.
+
+    Returns True if successful, False otherwise
+    """
+    resp = requests.post(f"{root_api_endpoint()}/workspaces/{workspace}")
+
+    if resp.ok:
+        return True
+
+    return False
+
+
+def create_graph(
+    workspace: str, graph_name: str, node_tables: List[str], edge_table: str
+) -> bool:
+    """Create a graph."""
+    resp = requests.post(
+        f"{root_api_endpoint()}/workspaces/{workspace}/graphs/{graph_name}",
+        params={"node_tables": node_tables, "edge_table": edge_table},
+    )
+
+    if resp.ok:
+        return True
+
+    return False
+
+
+def table_exists(workspace: str, table: str) -> bool:
+    """Check if the table exists."""
+    resp = requests.get(f"{root_api_endpoint()}/workspaces/{workspace}/tables/{table}")
+
+    if resp.status_code == 400:
+        return True
+    return False
+
+
+def create_table(workspace: str, table: str, data: str) -> bool:
+    """
+    Create table.
+
+    Returns True if successful, False otherwise
+    """
+    resp = requests.post(
+        f"{root_api_endpoint()}/csv/{workspace}/{table}", data=data.encode("utf-8")
+    )
+
+    if resp.ok:
+        return True
+    return False
+
+
+def log(text: str, indent: int):
+    """Log to console output."""
+    click.echo(click.wrap_text(text, initial_indent=(indent * " ")))
+
+
+@click.group()
+def cli():
+    """Script that helps with bootstrapping example data."""
+    pass
+
+
+@cli.command("populate")
+@click.argument("address", nargs=1, required=False)
+def populate(address: str):
+    """Populate arangodb with example data."""
+    global server_address
+    log_tabstop = 4
+    log_indent = 0
+
+    if address is not None:
+        address_parts = address.split(":")
+        address_parts = [x for x in address_parts if x]
+
+        if len(address_parts) != 2:
+            raise Exception("Invalid passed address.")
```

Comment:
Probably want a more detailed error message, something like

```suggestion
            raise Exception("Address must be of the form HOSTNAME:PORT")
```